### DISCUSSION TITLE
Response file is not found when running mock in docker container

### DISCUSSION THREAD

**Discussion by anapalmac** [mockoon/mockoon#857] [Help] [answered]
Hi,

I really like Mockoon's feature where I can specify a response file based on route parameter and, if the file is not found, the mock response is 404. It really helps the mock to behave organically to incoming requests.

Currently however I'm experiencing an issue that when I build a docker image for the mock and run it in a container, just locally, the response file is no longer found. I've tried different combinations of full and partial paths and the issue is still there: when I run the mock server in Mockoon desktop app, the response file is found, but when I run the docker container, I always get 404 response. 

Is this expected behavior? Am I doing something wrong?

I follow the instructions to create the docker image and run the container: https://mockoon.com/tutorials/run-mock-api-anywhere-cli/
And I specify the file path as such (among other variations tried):
![image](https://user-images.githubusercontent.com/110835674/199963276-4dd55588-b92a-4653-bea6-de89b1353333.png)

**Comment by anapalmac** [comment, marked as answer]
This stems from my lack of knowledge of docker: I just needed to copy the ResponseFiles folder from the local machine to the docker image using, for example, the COPY command. 
This could probably be done automatically as GitHub action (hopefully).

I leave the topic here anyway in case someone will face the same problem in the future.

**Reply by 255kb** [reply]
Glad you found the solution. 
Currently, the Docker image is not automatically adding `COPY` commands for the body files. 
It could be a nice improvement indeed.
